test/cloudflare-handler.test.ts

```diff
@@ -221,35 +221,26 @@ describe("createCloudflareHandler(app)", () => {
     const response = await handleRequest(request);
 
     expect(response.status).toEqual(200);
-    expect(await response.json()).toMatchInlineSnapshot(`
-      Object {
-        "authentication": Object {
-          "token": "scopedtoken456",
-        },
-        "data": Object {
-          "id": 1,
-        },
-      }
-    `);
```

Comment:
you can update the inline snapshots by running `jest` with `--updateSnapshot`

Reply:
Done.